### DISCUSSION TITLE
How to custom maxBatchOpenMs when version up to aws-sdk-java-v2

### DISCUSSION THREAD

**Discussion by lebkset** [aws/aws-sdk-java-v2#4528] [Q&A]
I custom maxBatchOpenMs in aws-sdk-java-v1 :
`@Configuration
public class SqsClientConfiguration {

    @Autowired(required = false)
    private AWSCredentialsProvider awsCredentialsProvider;

    @Autowired(required = false)
    private RegionProvider regionProvider;

    @Lazy
    @Bean(destroyMethod = "shutdown")
    public AmazonSQSBufferedAsyncClient amazonSQS() throws Exception {
        AmazonWebserviceClientFactoryBean<AmazonSQSAsyncClient> clientFactoryBean = new AmazonWebserviceClientFactoryBean<>(AmazonSQSAsyncClient.class,
                this.awsCredentialsProvider,
                this.regionProvider);
        clientFactoryBean.afterPropertiesSet();

        QueueBufferConfig bufferConfig = new QueueBufferConfig();
    	bufferConfig.setMaxBatchOpenMs(20);

        return new AmazonSQSBufferedAsyncClient(clientFactoryBean.getObject(), bufferConfig);
    }
}`

now, I migration to aws-sdk-java-v2 and want to custom **maxBatchOpenMs** , please suggest for me , thanks!

**Comment by debora-ito** [comment]
We don't have an equivalent for the `AmazonSQSBufferedAsyncClient` in v2 yet.

We are tracking the feature request here: https://github.com/aws/aws-sdk-java-v2/issues/165
